migrations/036-add_customers_table.js

```diff
@@ -0,0 +1,32 @@
+var db = require('./db')
+
+exports.up = function(next) {
+  const sql =
+    [
+    'CREATE TABLE customers ( ' +
+    'id uuid PRIMARY KEY,' +
+    'phone text unique NOT NULL,' +
+    'phone_at timestamptz NOT NULL,' +
```

Comment:
We might consider creating an "anonymous" record for every transaction that's recorded where we don't have any info. In some cases that will be most or all of the txs, so no point having a blank record here for every transaction.